.github/scripts/update-quality-gate-db.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Run your command and capture the output
+output=$(go run ./cmd/grype/main.go db list)
+
+# Extract the first instance of URL using grep, cut, and sed to trim leading whitespace
+url=$(echo "$output" | grep -m 1 -o 'URL: .*' | cut -d' ' -f2- | sed 's/^[[:space:]]*//')
+
+# Escape special characters in the URL for sed substitution
+escaped_url=$(printf '%s\n' "$url" | sed -e 's/[\/&]/\\&/g')
+
+# Replace TEST_DB_URL in specific Makefile using sed
+sed --e "s|^TEST_DB_URL = .*|TEST_DB_URL = $escaped_url|" test/quality/Makefile
```

Comment:
we could probably update the Makefile to read this from a file by default then this wouldn't need to be a sed command on the makefile... just overwrite the contents of a single file that only has the URL in question

Reply:
👍 Sure! I'll make it so we read from a file and update to use jq to reduce the complexity here